.travis.yml

```diff
@@ -17,11 +17,8 @@ env:
 - GO111MODULE=off
 go_import_path: github.com/nats-io/nats-server
 install:
-- go get github.com/nats-io/nats.go/
-- go get github.com/nats-io/nkeys
-- go get github.com/nats-io/jwt
-- go get -u honnef.co/go/tools/cmd/staticcheck
-- go get -u github.com/client9/misspell/cmd/misspell
+- GO111MODULE=on go get honnef.co/go/tools/cmd/staticcheck@2021.1.2
```

Comment:
Why do we need pinned versions of these again?

Reply:
So at first, when we were doing `go get -u`, staticcheck would fail because one of its dependency has a breaking change, so that made the whole thing fail with error like that:
```
# honnef.co/go/tools/go/loader
../../../honnef.co/go/tools/go/loader/loader.go:330:48: err.Line undefined (type config.ParseError has no field or method Line)
```
So I was looking at pinning to a version, but after thinking about it, if we do the `GO111MODULE on go get`, then it should be fine to not pin to a version, since this is using go.mod and by definition using pinned dependencies (assuming that each tool's main is not simply broken). So I will update the PR to remove the version.

Note, staticcheck's dependency resolved the issue yesterday (https://github.com/BurntSushi/toml/commit/dcab1c6c8e42cfd8d4ebc085554815678d7b1a2b), but still, using `GO111MODULE=on go get` seem to be the way to go, and allow us to pin to a version if one day something break in their main branch, so we don't have to wait for them to fix.

Reply:
With 1.17 we can remove vendor directory yes?

Reply:
Goal is at one point to do it. But need a bit more experimentation. I recall an issue with pulling tool deps (staticcheck/coverall, etc..) that would prevent running the project/tests with go mods. Will do that in another branch.